### PR TITLE
Code review for Coding Tracker

### PR DESCRIPTION
**Pull request by SinghxRaj**

### CODE REVIEW

File: SinghxRaj.CodingTracker/TimeFormat.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
```
Comment (TheCSharpAcademy) [inline SinghxRaj.CodingTracker/TimeFormat.cs:1]: Please remove unused using

File: SinghxRaj.CodingTracker/Display.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        Console.WriteLine("Welcome to Coding Tracker!");
+        Console.WriteLine("With Coding Tracker, you can " +
+                    "keep track of all your coding sessions.");
+        Console.WriteLine();
+    }
+
+    internal static void PrintOptions()
+    {
+        Console.WriteLine("Type 0 - Exit application");
```
Comment (TheCSharpAcademy) [inline SinghxRaj.CodingTracker/Display.cs:15]: Users should also be able to delete/update data, as per requirements: 
image

From the habit tracker project. 
image

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

File: SinghxRaj.CodingTracker/TimeFormat.cs (hunk 1 above)
Comment (SinghxRaj) [reply SinghxRaj.CodingTracker/TimeFormat.cs]: I removed the unused using directive.

File: SinghxRaj.CodingTracker/Display.cs (hunk 2 above)
Comment (SinghxRaj) [reply SinghxRaj.CodingTracker/Display.cs:15]: Added a delete coding session feature and updating coding session feature.

**Pull request merged**